.github/workflows/ci.yaml

```diff
@@ -18,6 +18,9 @@ jobs:
         matrix:
             os: [ubuntu-latest]
             python-version: ["3.10", "3.11"]
+            include:
+              - os: macos-latest
+                python-version: "3.10"
```

Comment:
Oh didn't know that was a option

Reply:
Same! I just learnt about it myself 🙂

Reply:
Will def. use it somewhere in the future thanks!